Emby.Server.Implementations/IO/ManagedFileSystem.cs

```diff
@@ -686,13 +686,16 @@ public virtual void DeleteFile(string path)
 
         public virtual List<FileSystemMetadata> GetDrives()
         {
-            // Only include drives in the ready state or this method could end up being very slow, waiting for drives to timeout
-            return DriveInfo.GetDrives().Where(d => d.IsReady).Select(d => new FileSystemMetadata
+            // check for ready state to avoid waiting for drives to timeout
+            // some drives on linux have no actual size or are used for other purposes
+            return DriveInfo.GetDrives().Where(d => d.IsReady)
+                .Where(d => d.TotalSize != 0)
```

Comment:
You could combine these to one Where-Statement with &&. e.g.: 
`Where(d => d.IsReady && d.TotalSize != 0 && d.DriveType != DriveType.Ram)`